packages/components/src/components/button-toggle-menu/button-highlight.jsx

```diff
@@ -3,8 +3,8 @@ import React     from 'react';
 
 const Highlight = ({ left, width }) => {
     const border_radius_size = '4px';
-    const left_offset        = (left < 110) ? 0 : left;
-    const width_offset       = (width < 110) ? 111 : width;
+    const left_offset        = (left < 60) ? 0 : left;
```

Comment:
What does the number 60 represent? can we put it in a constant?

Reply:
It seems that's minimum width of each highlight. I don't  know why previous number (110) exists at all! But changing it to 60 fixed the styling problem in p2p and didn't make any change in the previous usage of this component in the trader.
I think it doesn't have any special meaning to be in a separate variable.

Reply:
then i'd suggest this since it's easier to read:
```suggestion
    const left_offset        = (left < min_width) ? 0 : left;
```

Reply:
I removed the whole ternary part and everything works fine! 
```js
const left_offset   = left;
const width_offset   = width;
```

Reply:
@mahdiaryayi-fs even better 😄